src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -545,10 +551,12 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeLibrary Condition="'$(IlcLlvmExceptionHandlingModel)' == 'cpp'" Include="$(IlcSdkPath)libCppExceptionHandling.a" />
       <NativeLibrary Condition="'$(IlcLlvmExceptionHandlingModel)' == 'wasm'" Include="$(IlcSdkPath)libWasmExceptionHandling.a" />
       <NativeLibrary Condition="'$(IlcLlvmExceptionHandlingModel)' == 'emulated'" Include="$(IlcSdkPath)libEmulatedExceptionHandling.a" />
+      <_WASIComponentImports Include="$(IlcFrameworkNativePath)*.wit" />
```

Comment:
Are we ok mixing all caps, `WASI` and pascal, e.g. `Ilc` ?

Reply:
Changed to `Wasi`. It's intentionally similar to the item name used in `wasi.proj`.